docs/global/modules/core/pages/reference/component-mps-model-server-plugin.adoc

```diff
@@ -30,3 +30,9 @@ Reports an unhealthy system whenever no project is loaded.
 
 Reports an unhealthy system when no virtual folders are available.
 This might also be true in case a project without virtual folders is fully loaded.
+
+=== loadModels
+
+Can be used to eagerly load model data avoiding a slow first query.
+Multiple `moduleNamespace` parameters can be provided to specify the relevant modules.
+All modules starting with the string provided by that parameter are included.
```

Comment:
So an appropriate request would be?

```
/health?loadModels=true&moduleNamespace=foo.bar
```

A bit weird that parameters are now used for two purposes. One time for selecting the check, and then here as a special case for the modules.

In any case, a valid example call given this complexity would be good to have in the docs.

Reply:
I solved the part that requires MPS knowledge. Feel free to continue working on this PR.

Reply:
Ok. Thank you for solving the MPS parts. I can continue with the details.